processor/bin/process

```diff
@@ -222,6 +226,10 @@ function verify2() {
     function process() {
         local filename=$1
         export_data_dimensions "$filename"
+        if [[ $N_DATA == "null" ]]; then
+            echo "$BATCH_ID has unknown dimension, skipping..."
```

Comment:
Are these scripts run in production? Hopefully this won't generate too much output?

Reply:
Yes, this gets run in production. It shouldn't be too noisy (the number of lines is equal to the number of partitions), and should only occur during `verify1`.